### PR TITLE
sentry - add detection for Brave Browser

### PR DESCRIPTION
**Pull request by jennypollack**
- from Brave team: `window.chrome.ipcRenderer` is defined and can be used to identify Brave in background script
- adds flag to Sentry report to help detect chrome vs brave issues

### CODE REVIEW

File: app/scripts/background.js [hunk 1]
```diff
@@ -44,8 +45,9 @@ const notificationManager = new NotificationManager()
 global.METAMASK_NOTIFIER = notificationManager
 
 // setup sentry error reporting
-const release = platform.getVersion()
-const raven = setupRaven({ release })
+const releaseVersion = platform.getVersion()
+const raven = setupRaven({ releaseVersion })
+assert.equal('test', 'notest')
```
Comment (brunobar79) [inline app/scripts/background.js]: You might wanna remove this one (which is probably the one line that's breaking the e2e tests)

File: app/scripts/lib/setupRaven.js [hunk 2]
```diff
@@ -20,10 +22,13 @@ function setupRaven (opts) {
...
 
   const client = Raven.config(ravenTarget, {
-    release,
+    releaseVersion,
     transport: function (opts) {
+      opts.data.extra.isBrave = isBrave
       const report = opts.data
+
       try {
+        console.log('testing opts in transport sentry')
```
Comment (brunobar79) [inline app/scripts/lib/setupRaven.js]: and this one

**Review by brunobar79 — COMMENTED** [review]
LGTM! Clean up and ping me and I'll approve

File: app/scripts/background.js (hunk 1 above)
Comment (jennypollack) [reply app/scripts/background.js]: super weird, totally thought i removed this. thanks for taking a look.

**Review by brunobar79 — APPROVED** [review]
👍

**Pull request merged**